Tests/AI.cpp

```diff
@@ -93,6 +93,57 @@ TEST(ai_test, ai_is_winning){
     EXPECT_GT(val, 0);
 }
 
+TEST(ai_test, team_has_quaffle_works_for_left){
+    auto env = setup::createEnv();
+    auto leftTeam = env->getTeam(gameModel::TeamSide::LEFT);
+    env->quaffle->position = leftTeam->keeper->position;
+    auto res = ai::teamHasQuaffle(env, leftTeam->seeker);
+    EXPECT_EQ(res, true);
```

Comment:
Nutz doch das EXPECT_TRUE / EXPECT_FALSE
```suggestion
    EXPECT_TRUE(res);
```